Packs/Active_Directory_Query/ReleaseNotes/1_6_44.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Active Directory Query v2
+
+- Documentation and metadata improvements.
```

Comment:
```suggestion
Documentation and metadata improvements.
```
Do you need to be more specific?  Added 3268 for global catalog servers port value.